src/ripple/beast/test/yield_to.h

```diff
@@ -79,56 +75,51 @@ class enable_yield_to
         @param fn... One or more functions to invoke.
     */
 #if BEAST_DOXYGEN
-    template<class... FN>
+    template <class... FN>
     void
     yield_to(FN&&... fn)
 #else
-    template<class F0, class... FN>
+    template <class F0, class... FN>
     void
     yield_to(F0&& f0, FN&&... fn);
 #endif
 
-private:
-    void
-    spawn()
+        private : void spawn()
     {
     }
```

Comment:
A few lines above this one, the block:
```
#if BEAST_DOXYGEN
    template <class... FN>
    void
    yield_to(FN&&... fn)
#else
```
is missing the `;` on the function declaration. If you put it back, the formatting here won't look as janky.